ui/index.html

```diff
@@ -428,7 +428,7 @@ <h3 class="expansion-title w-100">Cataclysm</h2>
 										<img src="https://wow.zamimg.com/images/wow/icons/large/class_warlock.jpg" class="sim-link-icon border-warlock">
 										<div class="d-flex flex-column">
 											<span class="sim-link-title">Warlock</span>
-											<span class="launch-status-label text-brand">Not Yet Supported</span>
+											<span class="launch-status-label text-brand">Alpha</span>
```

Comment:
You also need to update `launched_sims.ts`

Reply:
Yeah